config/routes.rb

```diff
@@ -206,6 +206,12 @@
 
     namespace :web do
       resource :settings, only: [:update]
+      resource :push_subscriptions, only: [:create] do
+        # Why doesn't PUT /api/web/push_subscriptions/:id work?
```

Comment:
@Gargron Do you have some insights into this? If I call `axios.put('/api/web/push_subscriptions/2'` I get this in the logs:

```
Started PUT "/api/web/push_subscriptions/2" for 127.0.0.1 at 2017-06-24 16:34:57 +0200
Processing by ApplicationController#raise_not_found as HTML
```

`axios.put('/api/web/push_subscriptions')` works fine though.

Cannot seem to figure it out.